horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerRecyclerView.java

```diff
@@ -1,149 +1,159 @@
 package com.github.jhonnyx2012.horizontalpicker;
 
-import android.app.AlarmManager;
 import android.content.Context;
-import android.graphics.Color;
 import android.support.annotation.Nullable;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.LinearSmoothScroller;
 import android.support.v7.widget.LinearSnapHelper;
 import android.support.v7.widget.RecyclerView;
 import android.util.AttributeSet;
-import android.util.Log;
 import android.view.View;
 
-import org.joda.time.DateTime;
-import org.joda.time.Days;
+import org.threeten.bp.LocalDate;
+import org.threeten.bp.temporal.ChronoUnit;
 
 /**
  * Created by Jhonny Barrios on 22/02/2017.
- *
  */
 
 public class HorizontalPickerRecyclerView extends RecyclerView implements OnItemClickedListener, View.OnClickListener {
 
-    private HorizontalPickerAdapter adapter;
-    private int lastPosition;
-    private LinearLayoutManager layoutManager;
-    private float itemWidth;
-    private HorizontalPickerListener listener;
-    private int offset;
-
-    public HorizontalPickerRecyclerView(Context context) {
-        super(context);
-    }
-
-    public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs) {
-        super(context, attrs);
-    }
-
-    public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs, int defStyle) {
-        super(context, attrs, defStyle);
+  private final static int OFFSET = 3;
+  private HorizontalPickerAdapter adapter;
+  private LinearLayoutManager layoutManager;
+  private float itemWidth;
+  private HorizontalPickerListener listener;
+  private boolean scrollToSelectedPosition;
+
+  public HorizontalPickerRecyclerView(Context context) {
+    super(context);
+  }
+
+  public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs) {
+    super(context, attrs);
+  }
+
+  public HorizontalPickerRecyclerView(Context context, @Nullable AttributeSet attrs, int defStyle) {
+    super(context, attrs, defStyle);
+  }
+
+  public void init(Context context,
```

Comment:
Is there a way to add more inertia on the scroll view?   If I swipe, it scrolls very far in the list, doesn't seem to slow down

Reply:
Probably by playing with the fling manager or something, haven't had the time to look at it.